packages/multi-test/src/app.rs

```diff
@@ -354,17 +350,18 @@ where
     // this returns the contract address as well, so we can properly resend the data
     fn handle_wasm(
         &mut self,
-        sender: HumanAddr,
+        sender: Addr,
         msg: WasmMsg,
-    ) -> Result<(HumanAddr, ActionResponse<C>), String> {
+    ) -> Result<(Addr, ActionResponse<C>), String> {
         match msg {
             WasmMsg::Execute {
                 contract_addr,
                 msg,
                 send,
             } => {
+                let contract_addr = Addr::unchecked(contract_addr);
```

Comment:
I would validate here instead of unchecked.

`let contract_addr = self.api.addr_validate(&contract_addr)?;` should work here, well, except for the String error.

Maybe add `App.add_validate(addr: &str) -> Result<Addr, String>` which is a small wrapper around the above, then we can use that where needed in production code.